services/ansibler/server/ansible-playbooks/wireguard/templates/wg-dynamic.conf.j2

```diff
@@ -1,13 +1,21 @@
 [Interface]
+Address = {{ private_ip }}/24
 PrivateKey = {{ privatekey.stdout }}
 ListenPort = {{ wg_listen_port }}
 
-{% for host in groups['nodes'] %}
+{% for host in groups['dynamic'] %}
 {% if publickey.stdout != hostvars[host].publickey.stdout %}
 [Peer]
 PublicKey = {{ hostvars[host].publickey.stdout }}
 Endpoint = {{ hostvars[host].ansible_host }}:{{ wg_listen_port }}
 AllowedIps = {{ hostvars[host].private_ip }}/32
-PersistentKeepalive = 60
 {% endif %}
 {% endfor %}
+
+{% for host in groups['static'] %}
+{% if publickey.stdout != hostvars[host].publickey.stdout %}
+[Peer]
+PublicKey = {{ hostvars[host].publickey.stdout }}
```

Comment:
Do we not need `Endpoint = ...` here ?

Reply:
As these are static nodes, they do not have a publicly available endpoint.

Reply:
in the `wg-static.conf` the endpoint is set for static nodes

https://github.com/berops/claudie/blob/a4c5da5ad9c2733e7d1cddb16e530bc77ef1257c/services/ansibler/server/ansible-playbooks/wireguard/templates/wg-static.conf.j2#L20

Reply:
Yes, that's because we work under the assumption that static nodes can access each other over some private network.